### ISSUE TITLE
Port twisted.trial.reporter to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5964]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5964|
|Type|enhancement|
|Created|2012-09-12 13:32:09Z|

Trial has much nicer reporters than the stdlib unittest runner.  It would be nice to be able to use them on Python 3.

<details><summary>Searchable metadata</summary>

```
trac-id__5964 5964
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__navneet35371 navneet35371
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__ 
time__1347456729000000 1347456729000000
changetime__1439388744469387 1439388744469387
version__None None
owner__cpdean cpdean
cc__jml
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_3"></a>|@itamarst commented|
|-|-|

Not going to happen in minimal port.

**Comment by twisted-trac** [comment]
|<img alt="navneet35371's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|navneet35371 set owner to navneet35371<br>navneet35371 set status to `assigned`|
|-|-|

i am on to it..

**Comment by twisted-trac** [comment]
|<img alt="cpdean's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|cpdean commented|
|-|-|

I've started to look at this for pycon2014 sprints. I'm not sure how to contact navneet35371.

the tests for this module are in twisted.trial.test.test_reporter

**Comment by twisted-trac** [comment]
|<img alt="cpdean's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|cpdean commented|
|-|-|

twisted.trial.test.test_reporter  seems to rely on twisted.trial.runner ( [#5965](https://github.com/twisted/twisted/issues/5965) ).  going to jump on that now.

**Comment by twisted-trac** [comment]
|<img alt="cpdean's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|cpdean set owner to cpdean<br>cpdean set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_8"></a>|@hawkowl set status to `closed`|
|-|-|

Fixed in [#5965](https://github.com/twisted/twisted/issues/5965).
